test/test_kernels.py

```diff
@@ -525,7 +525,10 @@ def test_translations(ctx_factory, knl, local_expn_class, mpole_expn_class):
 
     del eval_offset
 
-    orders = [3, 4, 5]
+    if knl.dim == 2:
+        orders = [2, 3, 4]
```

Comment:
How come?

Reply:
4->5 is problematic for most of 2D kernels like 2->3 is problematic for 3D kernels.

Reply:
Fair enough. :shrug: